### DISCUSSION TITLE
Problem with LibreOffice and docx documents

### DISCUSSION THREAD

**Discussion by justbane** [pimcore/pimcore#8386] [Q&A / Support]
Having issues getting .docx files to load/show etc. in the DAM.

When I run ```bin/console maintenance``` I receive these errors

```Exception: Couldn't convert document to PDF: /local/htdocs/herman.local/web/var/assets/Intranet Assets/Documents/Service-Level-Agreement-HDMZ-Template-October-2017.docx with the command: '/usr/bin/soffice --headless --nologo --nofirststartwizard --norestore --convert-to pdf:writer_web_pdf_Export --outdir '/local/htdocs/herman.local/var/tmp' '/local/htdocs/herman.local/web/var/assets/Intranet Assets/Documents/Service-Level-Agreement-HDMZ-Template-October-2017.docx'' in /local/htdocs/herman.local/vendor/pimcore/pimcore/lib/Document/Adapter/LibreOffice.php:162```

I don't think it is a bug as I am not sure I have the right items installed maybe? Here is what I have:

```
Ghostscript 9.25
LibreOffice 5.3.6.1 30(Build:1)
```

When I run this command: ```/usr/bin/soffice --headless --nologo --nofirststartwizard --norestore --convert-to pdf:writer_web_pdf_Export --outdir '/local/htdocs/herman.local/var/tmp' '/local/htdocs/herman.local/web/var/assets/Intranet Assets/Documents/Service-Level-Agreement-HDMZ-Template-October-2017.docx'```

I get a simple error...

```Error: source file could not be loaded```

The file is verified to be there and the location is correct.

**Comment by brusch** [comment]
Libreoffice 5 seems to be quite outdated, maybe try it again again with a more up-to-date version?
